view/frontend/templates/footer/zopim.phtml

```diff
@@ -83,17 +78,17 @@
                                 z.set._ = [];
                                 $.async = !0;
                                 $.setAttribute("charset", "utf-8");
-                                $.src = "//v2.zopim.com/?<?php echo $block->config->getZopimId() ?>";
+                                $.src = "\/\/v2.zopim.com\/?<?= $block->escapeHtmlAttr($block->config->getZopimId()); ?>";
```

Comment:
I can't confirm if this will work since I we don't use this module on an active Magento 2 project

Reply:
We are using this activly in one of our customers sites right now. I can confirm it works.